roles/kibana/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+kibana_package_name: kibana
+kibana_service_name: kibana
+kibana_config_dir: /etc/kibana
+kibana_config_file: '{{ kibana_config_dir }}/kibana.yml'
+kibana_config_mode: 0644
+kibana_owner: kibana
+kibana_group: kibana
+
+# "0.0.0.0" to allow public access
+# "localhost" to allow access through proxy only
+kibana_server_bind: localhost
+kibana_server_port: 5601
+kibana_server_elasticsearch_url: 'http://{{ elasticsearch_bind }}:9200'
```

Comment:
Yes this is the same parameter. Right now there is only single elasticsearch client, which is Kibana, and target is kibana running on the same server as elasticsearch. Once we will support other architectures, we can surely iterate on that.